### PR TITLE
Updated prettier setup details

### PR DESCRIPTION
**Pull request by AlfieJones**
Although it's not this projects duty to inform the user how to install other dependencies, I've updated the eslint config prettier plugin setup. For the plugin to properly work you need to add the following rules 
`  "rules": {
    "indent": "error"
}`

You can find the relevant documentation here https://github.com/prettier/eslint-config-prettier

### CODE REVIEW

File: docs/basic-features/eslint.md [hunk 1]
```diff
@@ -190,11 +190,14 @@ npm install --save-dev eslint-config-prettier
...
 
-Then, add `prettier` to your existing ESLint config:
+Then, add `prettier` and the following `rules` to your existing ESLint config:
 
 ```json
 {
-  "extends": ["next", "prettier"]
+  "extends": ["next", "prettier"],
+  "rules": {
+    "indent": "error"
```
Comment (timneutkens) [inline docs/basic-features/eslint.md:199]: This is incorrect as per https://github.com/prettier/eslint-config-prettier#cli-helper-tool

> While the "prettier" config can disable problematic rules in "some-other-config-you-use", it cannot touch "rules"! (That’s how ESLint works – it lets you override configs you extend.) The CLI helper tool reports that "indent" conflicts with Prettier, so you can remove it. (Which is nice – simplifying your config!)

TLDR: `"indent": "error"` is an example of a rule that should be removed according to the readme.

Comment (AlfieJones) [reply docs/basic-features/eslint.md:199]: Yeah you're completely correct. Is it worth mentioning the inclusion of https://github.com/prettier/eslint-plugin-prettier and config `  "plugins": ["prettier"],
  "rules": {
    "prettier/prettier": "error"
  }`

Again it's outside of this projects scope but equally a common setup with prettier
